renterd failing to start after #1435

### Current Behavior

Running docker container on k3s kubernetes cluster. ARM64 nodes. Current dev (b377f76)

```
Using RENTERD_NETWORK environment variable
Using RENTERD_BUS_API_PASSWORD environment variable
Using RENTERD_DB_URI environment variable
Using RENTERD_DB_USER environment variable
Using RENTERD_DB_PASSWORD environment variable
Using RENTERD_DB_NAME environment variable
Using RENTERD_DB_METRICS_NAME environment variable
Using RENTERD_WORKER_ENABLED environment variable
Using RENTERD_AUTOPILOT_ENABLED environment variable
Using RENTERD_LOG_FILE_ENABLED environment variable
Using RENTERD_WORKER_API_PASSWORD environment variable
2024-08-22T18:34:28Z	INFO	renterd	{"version": "b377f76", "network": "zen", "commit": "b377f76", "buildDate": "2024-08-22T08:35:54Z"}
2024-08-22T18:34:28Z	INFO	sql	Using MySQL version 8.4.2
2024-08-22T18:34:29Z	INFO	api: Listening on [::]:9980
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa4de90]
goroutine 1 [running]:
main.(*node).Run(0x4000460008)
	/renterd/cmd/renterd/node.go:411 +0x210
main.main()
	/renterd/cmd/renterd/main.go:72 +0x3d8
```

same issue for 8a169ab, b2ff8d2, 82c7ad0 and some random builds from latest to 4ada222

```
Using RENTERD_NETWORK environment variable
Using RENTERD_BUS_API_PASSWORD environment variable
Using RENTERD_DB_URI environment variable
Using RENTERD_DB_USER environment variable
Using RENTERD_DB_PASSWORD environment variable
Using RENTERD_DB_NAME environment variable
Using RENTERD_DB_METRICS_NAME environment variable
Using RENTERD_WORKER_ENABLED environment variable
Using RENTERD_AUTOPILOT_ENABLED environment variable
Using RENTERD_LOG_FILE_ENABLED environment variable
Using RENTERD_WORKER_API_PASSWORD environment variable
2024-08-22T18:48:50Z	INFO	renterd	{"version": "4ada222", "network": "zen", "commit": "4ada222", "buildDate": "2024-08-15T15:16:54Z"}
2024-08-22T18:48:51Z	INFO	sql	Using MySQL version 8.4.2
2024-08-22T18:48:51Z	INFO	api: Listening on [::]:9980
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa235c4]
goroutine 1 [running]:
main.(*node).Run(0x4000438008)
	/renterd/cmd/renterd/node.go:411 +0x214
main.main()
	/renterd/cmd/renterd/main.go:72 +0x3d8
```

a bit older a945624 starts without any issue

### Expected Behavior

work

### Steps to Reproduce

1. Start renterd on docker arm64

### Version

anything between 4ada222 and b377f76

### What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?

debian 11, kubernetes

### Autopilot Config

x

### Bus Config

x

### Contract Set Contracts

x

### Anything else?

_No response_

--- a/cmd/renterd/node.go
+++ b/cmd/renterd/node.go
@@ -232,7 +232,7 @@ func newNode(cfg config.Config, network *consensus.Network, genesis types.Block)
 		}
 		setupFns = append(setupFns, fn{
 			name: "Autopilot",
-			fn:   func(_ context.Context) error { ap.Run(); return nil },
+			fn:   func(_ context.Context) error { go ap.Run(); return nil },
 		})
 		shutdownFns = append(shutdownFns, fn{
 			name: "Autopilot",
@@ -252,6 +252,7 @@ func newNode(cfg config.Config, network *consensus.Network, genesis types.Block)
 		setupFns:    setupFns,
 		shutdownFns: shutdownFns,
 
+		bus: bc,
 		cfg: cfg,
 
 		logger: logger.Sugar(),

